### DISCUSSION TITLE
Has the "Hide Breaking News Shelf" option really been removed?

### DISCUSSION THREAD

**Discussion by kineci** [ReVanced/revanced-discussions#1071] [Question] [answered]
While patching to the newest version, I encountered a notification stating that several patches have been removed, including the "Hide Breaking News Shelf" option. This feature is extremely important to me and one of the main reasons I use ReVanced. 

Before I proceed with updating the app, I would like to confirm whether this option has truly been removed, or if I just choosed the wrong patches or something.

Thanks in advance!
![Screenshot_20240910-194506~2](https://github.com/user-attachments/assets/393516a4-acea-4b7d-939b-b5b27bd309dd)

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
The patch was renamed, it was not removed.

This warning could be improved and it also show all patches added.

**Reply by kineci** [reply]
That's a relief. Thank you! Do you know what the feature has been renamed to?
